internal/expr/info.go

```diff
@@ -4,6 +4,19 @@ import (
 	"reflect"
 )
 
+// sqlair-provided M-type map
```

Comment:
Go comments should generally end with a full stop

Reply:
Done! Also added a full stop at the end of some other comments.